doc/sphinx/source/developer_guide/simulation/hitl.rst

```diff
@@ -0,0 +1,304 @@
+.. developer_guide simulation hitl
+
+.. _hitl:
+
+===========================
+Hardware in the Loop
+===========================
+
+Hardware In The Loop (HITL) simulation is a way to test an embedded system (the real hardware and software) by simulating its environment, 
+ie. sensor inputs, and comparing its output, ie. actuator outputs, to expected output values. 
+It is the closest to an actual flight without actually flying. Using Paparazzi's Software In The Loop (SITL) and HITL for validation of a 
+flight dynamics of a fixed wing UAV is in detail described in a paper "Software-and hardware-in-the-loop verification of flight dynamics 
+model and flight control simulationof a fixed-wing unmanned aerial vehicle" by Cal Coopmans and Michal Podhradsky. Refer to the paper for more details.
+
+Principle
+-----------
+
+While the SITL simulation executes the airborne code on the ground host, HITL is a way to run the autopilot code on the actual hardware in an 
+environment where sensors and actuators are simulated. The difference is shown in diagrams below: SITL generates the physical behavior of 
+the airplane from the Flight DynaMics (FDM) Block, then feeds the generated values into virtual sensors, the sensor inputs are processed in 
+the autopilot and the autopilot control ouputs are captured and fed back into the FDM. HITL does the same, except it communicates with the autopilot over serial ports.
+
+In HITL, two separate processes are involved:
+
+- The real autopilot code on the control board with its own IO (for example battery voltage, etc);
+- A flight model combined with a model of the actuators and the sensors.
+
+Commands computed by the autopilot are sent to the flight model which sends back simulated values of the sensors output.
+
+.. figure:: images/600px-Hitl.png
+
+  Hardware-in-the-loop (HITL) autopilot testing block diagram
+
+.. figure:: images/550px-Sitl.png
+
+  Software-in-the-loop (SITL) autopilot testing block diagram
+
+Limitations
+-----------------
+
+**Update March 2018: HITL currently works only with Pprzlink 1.0, please recompile paparazzi with PPRZLINK_LIB_VERSION=1.0. We are working on a fix.**
```

Comment:
I don't know if it is still true, but if yes, pprzlink v1 is not supported anymore. Maybe we can keep this page for now, but still add a warning that this might be broken.

Reply:
done